format-specs/geoparquet.md

```diff
@@ -1,5 +1,47 @@
+# Geospatial Parquet format
 
-## A Geospatial Parquet format
+## Overview
 
-This page is a scratch pad for ideas to define a 'geo-parquet' format, aligning on a default schema and format for geospatial data in Parquet.
+The [Apache Parquet][parquet] provides a standardized open-source columnar storage format. This specification defines how geospatial data
+should be stored in parquet format, including the representation of geometries and the required additional metadata.
 
+## Geometry columns
+
+Geometry columns are stored using the `BYTE_ARRAY` parquet type. They are encoded as WKB.
+
+## Metadata
+
+geoparquet files include additional metadata at two levels:
+
+1. File metadata indicating things like the version of this specification used
+2. Column (chunk) metadata with additional metadata for each geometry column
+
+## File metadata
+
+|     Field Name     |  Type  |                             Description                              |
+| ------------------ | ------ | -------------------------------------------------------------------- |
+| geoparquet_version | string | **REQUIRED** The version of the metadata standard used when writing. |
+| primary_column     | string | The "primary" geometry column.                                       |
+
+
+## Column metadata
+
+Each geometry column should include additional metadata
+
+| Field Name |                               Type                                      |                                                                   Description                                                                     |
+| ---------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
+| epsg       | integer\| null                                                          | **REQUIRED.** [EPSG code](http://www.epsg-registry.org/) of the datasource                                                                        |
```

Comment:
Mostly copied from https://github.com/stac-extensions/projection.

Do we want this here? Or do we keep the core spec lean and leave this to extensions?

Reply:
Sounds good.

> For the immediate term I think we just use epsg, and say it's required and must be set to 4326, and other options may be defined in extensions.

I vaguely recall discussions about non-geospatial geometries, i.e. no epsg / CRS. I don't recall if that was on the opening-call or another context, but should we allow `espg: null` to allow for this use case?